src/databricks/labs/remorph/cli.py

```diff
@@ -119,9 +118,9 @@ def transpile(
     status, errors = asyncio.run(do_transpile(ctx.workspace_client, engine, config))
 
     for error in errors:
-        print(str(error))
+        logger.error(f"Transpile error: {error}")
```

Comment:
IIRC the logging is done by do_transpile already
Now we want these errors in the console (this is the cli) without all the logged noise